lib/http-server.js

```diff
@@ -133,7 +133,18 @@ function createKeys (cb) {
 
       var opts = {
         days: 2048,
-        algorithm: 'sha256'
+        algorithm: 'sha256',
+        extensions: [
+          {
+            name: 'subjectAltName',
+            altNames: [
+              {
+                type: 2,
```

Comment:
could you add a comment here about what `type: 2` means?

Reply:
The resource I used for that is: https://github.com/digitalbazaar/forge/blob/master/lib/x509.js which has lots of great comments. Would you like me to include the url for more context?